ha/const.py

```diff
@@ -125,3 +125,66 @@ class NODE_STATUSES(Enum):
     STANDBY_WITH_RESOURCES_RUNNING = "Standby with resource(s) running"
     MAINTENANCE = "Maintenance"
     UNKNOWN = "Unknown"
+
+# System Health
+# Components health update hierarchy
+CLUSTER_UPDATE_HIERARCHY = ["cluster"]
+SITE_UPDATE_HIERARCHY = ["site"] + CLUSTER_UPDATE_HIERARCHY
+RACK_UPDATE_HIERARCHY = ["rack"] + SITE_UPDATE_HIERARCHY
+STORAGESET_UPDATE_HIERARCHY = ["storageset"] + RACK_UPDATE_HIERARCHY
+NODE_UPDATE_HIERARCHY = ["node"] + STORAGESET_UPDATE_HIERARCHY
+SERVER_UPDATE_HIERARCHY = ["server"] + NODE_UPDATE_HIERARCHY
+SERVER_HW_UPDATE_HIERARCHY = ["server_hw"] + SERVER_UPDATE_HIERARCHY
+SERVER_SERVICE_UPDATE_HIERARCHY = ["server_service", "agg_service"] + SERVER_UPDATE_HIERARCHY
+STORAGE_UPDATE_HIERARCHY = ["storage"] + NODE_UPDATE_HIERARCHY
+STORAGE_COMPONENT_UPDATE_HIERARCHY = ["storage_component"] + STORAGE_UPDATE_HIERARCHY
+
+# Constants
+RESOURCE_LIST = "resource_list"
+UPDATE_HIERARCHY = "update_hierarchy"
+
+# Components/resources maintained by System Health module. Note: Do not change the order.
+SYSTEM_HEALTH_COMPONENTS = {"server_hw": {RESOURCE_LIST: ["node:fru", "node:sensor", "node:os", "node:interface"],
```

Comment:
No. const.py should have constants only. I see, dictionaries and lists are being put in this file. It should be part of respective modules.

Reply:
sure, had started with a separate file under system_health initially and then ended-up moving it here :)

Reply:
done